modules/livestore/partition_reader.go

```diff
@@ -159,6 +161,25 @@ func (r *PartitionReader) commitLoop(ctx context.Context) {
 	}
 }
 
+func (r *PartitionReader) ownershipLoop(ctx context.Context) {
+	t := time.NewTicker(15 * time.Second)
+	defer t.Stop()
+
+	partition := fmt.Sprint(r.partitionID)
+	// initial set
+	r.metrics.ownedPartition.WithLabelValues(partition, r.consumerGroup).Set(1)
+
+	for {
+		select {
+		case <-ctx.Done():
+			r.metrics.ownedPartition.WithLabelValues(partition, r.consumerGroup).Set(0)
+			return
+		case <-t.C:
+			r.metrics.ownedPartition.WithLabelValues(partition, r.consumerGroup).Set(1)
+		}
```

Comment:
Does it need to be set on a ticker? Once it's initially set it should be in the registry until the live-store shuts down